src/modules/expression-evaluator/src/evaluateExpression.test.ts

```diff
@@ -451,20 +453,7 @@ test('Test Postgres get array of template names', () => {
 test('Test Postgres get Count of templates', () => {
   return evaluateExpression(testData.countTemplates, { pgConnection: pgConnect }).then(
     (result: any) => {
-      expect(result).toEqual(4)
-    }
-  )
-})
-
-test('Test Postgres get template names -- no type', () => {
-  return evaluateExpression(testData.getListOfTemplates_noType, { pgConnection: pgConnect }).then(
-    (result: any) => {
-      expect(result).toEqual([
-        { name: 'Test -- General Registration' },
```

Comment:
Sorry, this is not really for your PR, but can you update the tests that use this to the new name for the Demo template? (Currently 2 are failing due to this).

Reply:
Also, why delete this test? Is there a problem with it?

Reply:
It seemed to be exactly the same as `getListOfTemplates`

Reply:
Okay, it's actually testing a slightly different thing:
```
testData.getListOfTemplates = {
  type: 'array',
  operator: 'pgSQL',
  children: [
    {
      value: 'SELECT name FROM template',
    },
  ],
}
```

vs

```
testData.getListOfTemplates_noType = {
  operator: 'pgSQL',
  children: [
    {
      value: 'SELECT name FROM template',
    },
  ],
}
```

The "type" parameter (which is not used much, but can be important) causes the query to return a different shaped array, so I think we should keep it in. In fact, I think I made these 2 different tests in order to demonstrate the difference.

From the docs:
> The type property is optional in most cases, but some operators will return their results in a different format depending on the type (e.g. pgSQL -- see below).

Reply:
Oh, I see. Sorry. Will revert that removal now : )

Reply:
Done: be26805